test/InputOutput/ArtifactWrappers/runtests.jl

```diff
@@ -0,0 +1,35 @@
+module TestArtifactWrappers
+
+using Test
+using ClimateMachine.ArtifactWrappers
+
+@testset "ArtifactFile" begin
+    af = ArtifactFile(url = "url", filename = "filename")
+    @test af.url == "url"
+    @test af.filename == "filename"
+end
+
+@testset "ArtifactWrapper" begin
+    mktempdir() do path
+        af = ArtifactFile(
+            url = "https://httpbin.julialang.org/ip",
```

Comment:
@kpamnany I grabbed this from Julia Base, not sure the best way to test something that uses `Base.download` 😄

Reply:
you can download something locally with `file://`

ex. create a temp file with known content (so the hash is the same)
`file:///tmp/foo.txt`

Reply:
I don't know if this work on windows but it should, and will once Base has libcurl included in 1.6 :)

Reply:
Oh cool, I'll try that, thanks!

Reply:
Done